addon/theming.js

```diff
@@ -0,0 +1,88 @@
+// Create a dummy theme profile based on the dark theme
+// NOTE: This should not be required once
+// https://bugzilla.mozilla.org/show_bug.cgi?id=1435216 is addressed
+const theme = {
+  colors: {
+    accentcolor: "#4A4A4F",
+    textcolor: "#fff",
+    popup: "#4A4A4F",
+    popup_text: "#fff",
+    popup_highlight: "#6D6D6F",
+    popup_highlight_text: "#fff"
+  }
+};
+
+const updateBackground = function(el, color) {
+  el.style.background = color;
+};
+
+const updateColor = function(el, color) {
+  el.style.color = color;
+};
+
+function updateSidebarStyle() {
+  if (theme.colors) {
+    let body = document.getElementsByClassName("page")[0];
+    let links = document.getElementsByTagName("buttons");
+
+    let accentcolor = theme.colors.accentcolor || theme.colors.frame;
+
+    if (accentcolor) {
+      updateBackground(document.body, accentcolor);
+      updateBackground(body, accentcolor);
+    }
+
+    if (theme.colors.textcolor) {
+      updateColor(document.body, theme.colors.textcolor);
+      for (let i = 0; i < links.length; i++)
+        updateColor(links[i], theme.colors.textcolor);
+    }
+  }
+}
+
+function updatePopupStyle() {
+  if (theme.colors) {
+    let panel = document.getElementById("panel");
+    let tabs = document.getElementsByClassName("tab");
+    let feedback = document.getElementsByClassName("feedback-button")[0];
```

Comment:
document.getQueryselector('feedback-button') will get you only the first element it finds with that selector